src/components/Reports/Reports.jsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Container,
+  Box,
+  Heading,
+  IconButton,
+  Icon,
+  Text,
+  Link,
+} from "@chakra-ui/react";
+import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
+import { GoArrowLeft } from "react-icons/go";
+import css from "./Reports.module.css";
+import ExpensesAndIncome from "./ExpensesAndIncome";
+import { useState } from "react";
+
+const Reports = () => {
+  const [currentDate, setCurrentDate] = useState(new Date());
+
+  const handleNextMonth = () => {
+    const nextMonth = new Date(currentDate);
+    nextMonth.setMonth(nextMonth.getMonth() + 1);
+    setCurrentDate(nextMonth);
+  };
+
+  const handlePrevMonth = () => {
+    const prevMonth = new Date(currentDate);
+    prevMonth.setMonth(prevMonth.getMonth() - 1);
+    setCurrentDate(prevMonth);
+  };
+  const currentPeriod = currentDate.toLocaleString("default", {
+    month: "long",
+    year: "numeric",
+  });
+
+  return (
+    <>
+      <Container className={css.first_container}>
+        <Box className={css.button_box}>
+          <Link className={css.back_icon_box}>
+            <IconButton
+              className={css.back_arrow}
+              variant="link"
+              colorScheme="orange"
+              aria-label="Back to main page"
+              icon={<Icon as={GoArrowLeft} />}
+            ></IconButton>
+            <Text className={css.back_to_main_text}>Main page</Text>
+          </Link>
+          <Box className={css.month_box}>
```

Comment:
to tez jest w komponencie bilansu, a dokladniej MonthsPaginator, można tam przesunac dzialanie paginatora, bo poki co jest czysta stylizacja